Dalamud/Interface/FontAwesome/FontAwesomeIcon.cs

```diff
@@ -3375,20 +2619,6 @@ public enum FontAwesomeIcon
     [FontAwesomeCategoriesAttribute(new[] { "Maps" })]
     Directions = 0xF5EB,
 
-    /// <summary>
-    /// The Font Awesome "discord" icon unicode character.
-    /// </summary>
-    [Obsolete]
-    [FontAwesomeSearchTerms(new[] { "discord" })]
-    Discord = 0xF392,
```

Comment:
Do we want to make an exception to keep the Discord icon, i.e. for Discord links within plugins?

Reply:
None of these icons actually render today (including Discord). Font Awesome includes them in a separate "brands" file. This was excluded to keep the atlas manageable.

Reply:
Ah, got it!